controllers/machinedeployment_controller.go

```diff
@@ -203,7 +206,27 @@ func (r *MachineDeploymentReconciler) reconcile(ctx context.Context, cluster *cl
 		if d.Spec.Strategy.RollingUpdate == nil {
 			return ctrl.Result{}, errors.Errorf("missing MachineDeployment settings for strategy type: %s", d.Spec.Strategy.Type)
 		}
-		return ctrl.Result{}, r.rolloutRolling(ctx, d, msList)
+
```

Comment:
I assume it's okay (same as for KCP) that it could be that this code is only run every 10 minutes (default syncPeriod). This doesn't matter because rolloutAfter is set to a much larger value then 10m and then we don't care about a few minutes, right?

Increasing the syncPeriod would lead to a higher delay (on average) for rolloutAfter?